tests/pipelines/test_pipelines_zero_shot_image_classification.py

```diff
@@ -172,6 +172,22 @@ def test_small_model_tf(self):
             ],
         )
 
+    @slow
+    @require_torch
+    def test_large_model_misconfigured(self):
+        # XXX this should be a fast test, but the triggering arch
+        # VisionTextDualEncoderModel is missing for small tests
+        # https://huggingface.co/hf-internal-testing
+        # This test will also start to fail, once this architecture
+        # correctly defines AutoFeatureExtractor. At this point
+        # we can safely remove this test as we don't really want
+        # to keep around an invalid model around just for this.
+        with self.assertRaises(EnvironmentError):
+            pipeline(
+                task="zero-shot-image-classification",
+                model="Bingsu/vitB32_bert_ko_small_clip",
+            )
```

Comment:
I wouldn't include this test as it is. It's easy to build a small model in hf-internal-testing that is misconfigured and does not provide a tokenizer class while using a generic encoder/decoder arch.

Reply:
True.